src/dht.js

```diff
@@ -9,19 +9,29 @@ module.exports = (node) => {
 
       node._dht.put(key, value, callback)
     },
-    get: (key, callback) => {
+    get: (key, maxTimeout, callback) => {
```

Comment:
things like timeouts should be in a options object rather than a arg otherwise you will fall into a future of having a huge array of options. @vasco-santos @jacobheun

Reply:
You are right! I will create a new PR for changing it here, as well as for `js-libp2p-kad-dht` and will change `js-ipfs` too.